spec/features/participants_spec.rb

```diff
@@ -13,7 +13,6 @@
     @application_letter = FactoryGirl.create(:application_letter_accepted, user: @user, event: @event)
     visit event_participants_path(@event)
     select I18n.t("participant_groups.options.#{ParticipantGroup::GROUPS[9]}"), from: "participant_group[group]", match: :first, visible: false
-    expect(page).to have_text(I18n.t("participant_groups.update.successful"))
```

Comment:
since we're updating via ajax, we no longer display a text upon changes to the colors.